apps/client/src/Routes/Blog/PostPage/PostDetails/Display.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { BrowserRouter } from "react-router-dom";
+
+import { BlogPost } from "Api/BlogPost";
+import PostDetails from "./Display";
+
+const post: BlogPost = {
+  id: 1,
+  title: "Teste de blog post",
+  description: "Isso é um teste",
+  created_at: "2021-06-29T23:50:54.596Z",
+  image: {
+    url:
+      "https://images.pexels.com/photos/8285483/pexels-photo-8285483.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260",
+    name: "imagem aleatória",
+  },
+  content: "Esse é o conteúdo do post!",
```

Comment:
eu acho importante a gente testar todas as variações de estilo que o usuário criador de conteúdo pode usar aqui no storybook. as coisas que o usuário vai poder usar do markdown são:

- [ ] parágrafos
  - [ ] texto normal
  - [ ] texto em itálico
  - [ ] text negrito
  - [ ] links normais
- [ ] imagens
- [ ] links do youtube (renderiza o player em vez de mostrar um link normal)
- [ ] listas ordenadas*
- [ ] listas não-ordenadas*

*se o @Jaxiii decidir a gente pode jogar fora essas em decidir não implementar. aí eu só preciso avisar os usuário pra não usar elas nos posts

Reply:
Eu adicionei um texto no storybook que testa as variações de estilo que tinham no layout no figma. Acho que eram todas essas de cima, mas se faltar alguma é só me avisar que eu incluo.